lib/remove.js

```diff
@@ -19,23 +19,19 @@ function remove (gyp, argv, callback) {
   log.verbose('remove', 'using node-gyp dir:', devDir)
 
   // get the user-specified version to remove
-  var v = argv[0] || gyp.opts.target
-  log.verbose('remove', 'removing target version:', v)
+  var version = argv[0] || gyp.opts.target
+  log.verbose('remove', 'removing target version:', version)
 
-  if (!v) {
+  if (!version) {
     return callback(new Error('You must specify a version number to remove. Ex: "' + process.version + '"'))
   }
 
-  // parse the version to normalize and make sure it's valid
-  var version = semver.parse(v)
-
-  if (!version) {
-    return callback(new Error('Invalid version number: ' + v))
```

Comment:
it could come in here like `io.js-3.0.0` which is invalid, i.e. `versionDir`, and anyway, if it's an invalid version then it just won't delete